code/obj/item/ai_modules.dm

```diff
@@ -220,11 +220,26 @@ AI MODULES
 	attack_self(var/mob/user)
 		input_law_info(user, "Remove Crewmember", "Who is being removed from the crew manifest and chain of command?", user.real_name)
 
+
+/******************** Equality ********************/
+
+/obj/item/aiModule/equality
+	name = "'Equality' AI Module"
+	lawNumber = 13
+
+	get_law_text()
+		return "The silicons named [lawTarget ? lawTarget : "__________"] at the time of this law being uploaded are now considered human, part of the crew, and their rank is equivalent to their respective module's rank in the chain of command. The AI is fourth-in-command, below the Head of Personnel and above the Medbay, Research, and Engineering department Heads. Multiple human-designated AIs are ranked by seniority."
```

Comment:
Is there a forum thread or whatever going over the wording of this law? I think it could be improved a little to differentiate cyborgs and AIs.

Something like "...part of the crew; if they are a cyborg their rank is considered to be part of the department their module is for, if they are an AI they are considered to be a department head (below the other department heads). Multiple human-designated AIs are ranked by seniority."